pages/experiments/[id].tsx

```diff
@@ -0,0 +1,37 @@
+import debugFactory from 'debug'
+import { useRouter } from 'next/router'
+import React, { useEffect, useState } from 'react'
+import Container from 'semantic-ui-react/dist/commonjs/elements/Container'
+
+import AnalysesApi from '@/api/AnalysesApi'
+
+import ErrorsBox from '@/components/ErrorsBox'
+import Layout from '@/components/Layout'
+
+import { Analysis } from '@/models'
+
+const debug = debugFactory('abacus:pages/index.tsx')
+
+export default function ExperimentPage() {
+  const experimentId = Number(useRouter().query.id)
+  debug(`ExperimentPage#render ${experimentId}`)
+
+  const [error, setError] = useState<Error | null>(null)
+  const [analyses, setAnalyses] = useState<Analysis[] | null>(null)
+
+  useEffect(() => {
+    AnalysesApi.findByExperimentId(experimentId)
+      .then((analyses) => setAnalyses(analyses))
+      .catch(setError)
```

Comment:
Could be simplified to also handle when the id changes and the error needs to be reset also needs to handle when `experimentId` is `null` or `undefined` and maybe if `NaN` since you attempt to cast to `Number` above. Can easily just go to `/path/to/analsyis/not-a-number` to cause it.

```suggestion
  useEffect(async () => {
    if(experimentId == null) return;
    
    let analysis = error = null;
    try {
      analysis = await AnalysisApi.findExperimentId(experimentId)
    } catch (er) {
      error = er
    } finally {
      setAnalysis(analysis)
      setError(error);
    }
```

or, at least

```suggestion
  useEffect(() => {
    if(experimentId == null) return;
    AnalysesApi.findByExperimentId(experimentId)
      .then((analyses) => {
        setAnalyses(analyses)
        setError(null)
      })
      .catch((er) => {
        setError(er)
        setAnalysis(null)
      })
```